launchCronAndLogToMXDev.sh

```diff
@@ -0,0 +1,12 @@
+
+MXDEV_URL=$(cat ~/MXControl/config/MXDEV_URL)
+
+function postToLambda {
+    message="$(echo $1 | sed "s/^.*]: /**AUTO: **/")"
+    curl --silent -X POST -d "{\"source\": \"MXControl\",\"message\":\"$message\"}" "$MXDEV_URL"
```

Comment:
REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Reply:
what? it's in a function :P

Reply:
the ree is about the JSON >:c